gerritbot/src/lib.rs

```diff
@@ -615,6 +615,12 @@ where
 
         let is_human = !approver.to_lowercase().contains("bot");
 
+        // filter all messages that were already sent to the user recently
```

Comment:
I guess this is not true anymore.

Reply:
This will be true again with a following change that combines multiple approvals into a single message.